uber/models.py

```diff
@@ -1252,6 +1263,9 @@ class Attendee(MagModel, TakesPaymentMixin):
     ec_phone      = Column(UnicodeText)
     cellphone     = Column(UnicodeText)
 
+    # Represents a request for hotel booking info during preregistration
+    requested_hotel_info = Column(Boolean, default=False)
```

Comment:
We may want to actually make this True by default -- that way, when we import staff, they will automatically be emailed a link even if they don't claim their badge by the time we send the link out.